src/state-calculator-form.tsx

```diff
@@ -81,6 +85,53 @@ const renderUtilityField = (
       loading={utilitiesFetch.state === 'loading'}
     />
   );
+
+  let gasSelector = null;
+  const gasUtilities =
+    utilitiesFetch.state === 'complete'
+      ? utilitiesFetch.response.gas_utilities
+      : utilitiesFetch.state === 'loading'
+      ? utilitiesFetch.previousResponse?.gas_utilities
+      : null;
+
+  // If the response does not contain the gas utilities key, that means the
+  // user's gas utility is irrelevant, so we don't show the selector. If the key
+  // is present, we need to show the selector even if the gas utilities map is
+  // empty, because it matters whether the user actually has no gas service, or
+  // has gas service that's not listed here (indicated by the Other item).
+  if (gasUtilities) {
+    const gasOptions = Object.entries(gasUtilities)
+      .map(([id, info]) => ({ value: id, label: info.name }))
+      .concat([
+        {
+          value: DELIVERED_FUEL_UTILITY_ID,
+          label: msg('Delivered propane or fuel oil'),
+        },
+        { value: NO_GAS_UTILITY_ID, label: msg('No gas service') },
+        { value: OTHER_UTILITY_ID, label: msg('Other') },
+      ]);
+    gasSelector = (
+      <Select
+        id="gas_utility"
+        labelText={msg('Gas Utility', { desc: 'as in utility company' })}
+        tooltipText={msg('Choose the company you pay your gas bill to.')}
```

Comment:
(no change required) "to which you pay your gas bill" if probably more grammatically correct but also overly formal, hmmm...